docs/root/intro/version_history.rst

```diff
@@ -44,6 +44,7 @@ Version history
   to trigger health check response. Deprecated the
   :ref:`endpoint option <envoy_api_field_config.filter.http.health_check.v2.HealthCheck.endpoint>`.
 * health check: added support for :ref:`custom health check <envoy_api_field_core.HealthCheck.custom_health_check>`.
+* health check: health check connections can now be configured to use http/2
```

Comment:
nit: period end of sentence (feel free to fix others close by where we are not consistent).